### PR TITLE
EHT-276 Update Hedera JSON-RPC Relayer Instructions

### PR DESCRIPTION
**Pull request by kowshikvajrala**

### CODE REVIEW

File: src/content/docs/validator/external-chains/hedera.mdx [hunk 1]
```diff
@@ -71,8 +77,54 @@ importer-1  | 2024-09-05T14:10:44.539Z  INFO scheduling-4 c.h.m.i.d.r.RecordFile
...
+
+OPERATOR_KEY_MAIN=<DER Encoded Private Key, can be obtained from portal.hedera.com, wallet or exchange of choice>
+```
+
+### Start the RPC relay server
+
+```bash
+npm run start
+```
+
```
Comment (aditya-manit) [inline src/content/docs/validator/external-chains/hedera.mdx]: Can we please run it as as ervice

**Review by aditya-manit — CHANGES_REQUESTED** [review]
lets run it as a service

File: src/content/docs/validator/external-chains/hedera.mdx [hunk 2]
```diff
@@ -111,10 +111,27 @@ OPERATOR_ID_MAIN=<AccountID e.g. 0.1.2, can be obtained from portal.hedera.com,
...
+```bash 
+[Unit]
+Description=Hedera JSON-RPC Relayer
+After=network.target
+
+[Service]
+Type=simple
+User=root
```
Comment (aditya-manit) [inline src/content/docs/validator/external-chains/hedera.mdx]: $USER or hedera

File: src/content/docs/validator/external-chains/hedera.mdx [hunk 3]
```diff
@@ -111,10 +111,27 @@ OPERATOR_ID_MAIN=<AccountID e.g. 0.1.2, can be obtained from portal.hedera.com,
...
+WorkingDirectory=/path/to/hedera-json-rpc-relayer
+ExecStart=/bin/bash -c 'export NVM_DIR="/root/.nvm" && [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" && npm run start'
```
Comment (aditya-manit) [inline src/content/docs/validator/external-chains/hedera.mdx]: We can export NVM_DIR="/root/.nvm in Environment too I think

Comment (aditya-manit) [inline src/content/docs/validator/external-chains/hedera.mdx]: can $NVM_DIR/nvm.sh be added in ExecStartPre ?

**Review by aditya-manit — CHANGES_REQUESTED** [review]
__

File: src/content/docs/validator/external-chains/hedera.mdx (hunk 3 above)
Comment (kowshikvajrala) [reply src/content/docs/validator/external-chains/hedera.mdx]: tried not working with ExecStartPre

**Review by aditya-manit — APPROVED** [review]

**Pull request merged**